pkgs/development/lua-modules/overrides.nix

```diff
@@ -273,9 +276,21 @@ with super;
      sed -i 's,\(option(WITH_SHARED_LIBUV.*\)OFF,\1ON,' CMakeLists.txt
      rm -rf deps/libuv
     '';
+  });
+
+  luv-dev = super.luv.override({
```

Comment:
could you add a comment explaining the difference between luv and luv-dev ? why it's not luv.dev output for instance

Reply:
cc @gloaming 
I assume this is not a `luv.dev` output, since a different build is necessary here

Reply:
It was the case when I had tried building luv but maybe they improved it by now.
If it's a temporary package to work around luv "faulty" build system, it would be good to mention it